### PR TITLE
Update NEG usage metrics emit.

### PR DESCRIPTION
**Pull request by sawsa307**
* Update NEG usage metrics so it is now emitted via NEG metrics collector.

/assign @swetharepakula

### CODE REVIEW

File: pkg/metrics/features.go [hunk 1]
```diff
@@ -101,18 +101,6 @@ const (
 	sslPolicy      = feature("SSLPolicy")
```
Comment (swetharepakula) [inline pkg/metrics/features.go:101]: line 81, the feature "NEG" does that need to be removed?

Comment (sawsa307) [reply pkg/metrics/features.go:101]: No we need that for servicePort metrics at line 286

Comment (swetharepakula) [reply pkg/metrics/features.go:101]: or is this only used for Ingress metrics?

Comment (sawsa307) [reply pkg/metrics/features.go:101]: It is used for both Ingress and NEG metrics.
In ingress, we used it as the label for features on a service port.
In NEG, we used it as the label for neg usage metrics.

Comment (swetharepakula) [reply pkg/metrics/features.go:101]: clarified offline. NEG is used by multiple metrics, so we will duplicate this one.

**Review by swetharepakula — COMMENTED** [review]
/lgtm
/approve

**Comment by sawsa307** [comment]
image

Local run result to verify metrics are transported correctly.

**Pull request merged**